packages/@aws-cdk/toolkit-lib/lib/actions/diff/index.ts

```diff
@@ -114,4 +115,32 @@ export interface DiffOptions {
    * @default false
    */
   readonly securityOnly?: boolean;
+
+  /**
+   * Used a template from disk instead of from the server
+   *
+   * @default - Use from the server
+   */
+  readonly templatePath?: string;
+
+  /**
+   * Stack parameters for CloudFormation used at deploy time
+   * @default StackParameters.onlyExisting()
+   */
+  readonly parameters?: StackParameters;
+
+  /**
+   * Whether to run the diff against the template after the CloudFormation Transforms inside it have been executed
+   * (as opposed to the original template, the default, which contains the unprocessed Transforms).
+   *
+   * @default false
+   */
+  readonly compareAgainstProcessedTemplate?: boolean;
+
+  /**
+   * Whether or not to create, analyze, and subsequently delete a changeset
+   *
+   * @default true
+   */
+  readonly changeSet?: boolean;
```

Comment:
need help with a decision here: the CLI default is `true`, and that is enforced via yargs default -- the code itself actually does not default to true (i.e. it is `if (options.changeSet) { ... }`). because of this, currently in this PR, programmatic toolkit updates that conditional to `if (options.changeSet ?? true)` to behave the same way.

Reply:
I'm guessing in the CLI this default is set via the cli option.